school_library_decorator/person.rb

```diff
@@ -4,7 +4,7 @@ class Person
   attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(parent_permission, name = 'Unknown', age = 0)
```

Comment:
- [ ] The same error happens here as well. Please update the parameter of this method so it can better reflect the arguments passed in when you call this method.

Reply:
At the same moment, you are passing in name, age but the order here is parent_permission,name and age, so please pass in the additional argument parent_permission or set default value for parent_permission

Reply:
Thank you for this valuable review. ❤️ 

The issue has been resolved in the latest comment [e10783f](https://github.com/AshakaE/mini-oop/commit/e10783f134fc51496033e8e21c7f38c824532177). 🟢